CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Changed
+
+- Rework hooks.
```

Comment:
IMO the changelog entry should be more descriptive towards what has changed and maybe why it was neccessary. In the sense of: What is the improvement introduced.